spec/io/sql_data_source_spec.rb

```diff
@@ -14,70 +14,58 @@
     ActiveRecord::Base.connection
   end
 
+  let(:dat_file) do
+    'spec/fixtures/bank2.dat'
+  end
```

Comment:
And same about all others (dbi_handle, active_record_connection). In fact, you can just do in an appropriate context:
```ruby
let(:source) { DBI.connect("DBI:SQLite3:#{db_name}") }
```
...and that's it.